src/openms/source/ANALYSIS/OPENSWATH/MRMDecoy.cpp

```diff
@@ -537,9 +537,9 @@ namespace OpenMS
     // Go through all peptides and apply the decoy method to the sequence
     // (pseudo-reverse, reverse or shuffle). Then set the peptides and proteins of the decoy
     // experiment.
-    for (Size i = 0; i < exp.getPeptides().size(); i++)
+    for (Size pep_it = 0; pep_it < exp.getPeptides().size(); ++pep_it)
```

Comment:
also I believe that naming the variable about regarding to which vector it indexes makes more sense than just `i`, but maybe `pep_idx` makes more sense than `pep_it` - what do you think?